README.md

```diff
@@ -1,34 +1,31 @@
 # AnyCable Next.js/Vercel Example
 
-This is an example of how to use [AnyCable](https://anycable.io) with [Next.js](https://nextjs.org) and [Vercel](https://vercel.com) to build real-time applications.
-
 [![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fanycable%2Fvercel-anycable-demo&env=CABLE_URL,ANYCABLE_HTTP_BROADCAST_URL,ANYCABLE_HTTP_BROADCAST_SECRET,ANYCABLE_JWT_ID_KEY&envDescription=Link%20Vercel%20application%20with%20AnyCable%20server&envLink=https%3A%2F%2Fgithub.com%2Fanycable%2Fvercel-anycable-demo&project-name=vercel-anycable-demo&repository-name=vercel-anycable-demo)
 
-Learn more about AnyCable for serverless JavaScript apps in [the documentation](https://docs.anycable.io/guides/serverless).
-
-## Deployment
+This is an example of how to use [AnyCable](https://anycable.io) with [Next.js](https://nextjs.org) and [Vercel](https://vercel.com) to build real-time applications.
 
-### Prerequisites
+Learn more about AnyCable for serverless JavaScript apps in [the documentation](https://docs.anycable.io/guides/serverless).
 
-You need to deploy AnyCable-Go server to the platform of your choice. For a quick start, we recommend using [Fly.io](https://fly.io) and provide an example configuraiton.
+## Prerequisites
 
-> [!NOTE]
-> AnyCable-Go can be deployed to any platform. We use Fly.io in this example because it's easy to deploy and manage.
+You will need to deploy an AnyCable-Go server to the platform of your choice. We recommend using [Fly.io](https://fly.io) and provide an example configuration in `fly.toml.example` for seamless deployments. However, Anycable-Go server can be deployed on any platform.
```

Comment:
Removed all > [!NOTE] references. The templates page pulls in your README and these do not render correctly.